package.json

```diff
@@ -164,22 +164,27 @@
                 },
                 {
                     "command": "aws.login",
+                    "when": "view == lambda",
```

Comment:
Just making sure:
* Can we have multiple conditions? Say we want to have three or four different sub-panes, will this be usable by all of them?
* What happens if someone else makes a panel with view == lambda?

Reply:
Yes, we can have more complicated conditionals.

I've been thinking about the view names question too. It is in our best interest to make a more unique panel name. Should I do that in this PR?

Reply:
Sounds good. Maybe prefix the names with `aws.` (or something similar)? Then we could use `view.includes("aws.")` (assuming this field takes javascript)

Reply:
When clauses are not well documented. They do not accept javascript.
Here is what _is_ documented: https://code.visualstudio.com/docs/getstarted/keybindings#_when-clause-contexts

Reply:
It supports regex! https://code.visualstudio.com/docs/getstarted/keybindings#_keyvalue-when-clause-operator

Reply:
We use regex as an example a couple lines below this change to restrict when lambda context menu items appear